src/scripts/migrate.py

```diff
@@ -43,7 +44,24 @@ def slurp(
         owned_prefix_map: Dict[PREFIX, URI] = onto_config['base_prefix_map']
     sssom_df: pd.DataFrame = pd.read_csv(mondo_mappings_path, comment='#', sep='\t')
     excluded_terms: Set[CURIE] = get_excluded_terms(onto_exclusions_path)
-    mondo_term_ids: Set[int] = get_mondo_term_ids(mondo_terms_path, slurp_dir_path)
+
+    # Get next_mondo_id
+    next_mondo_id = min_id
+    slurp_files = [x for x in os.listdir(slurp_dir_path) if x.endswith('.tsv')]
+    for f in slurp_files:
+        slurp_df = pd.read_csv(os.path.join(slurp_dir_path, f), sep='\t')
+        slurp_ids = [int(x.split(':')[1]) for x in list(slurp_df['mondo_id'])[1:]]
+        next_mondo_id = max(next_mondo_id, max(slurp_ids) + 1)
+
+    # Get map of native IDs to existing slurp mondo IDs
+    slurp_id_map: Dict[str, str] = {}
```

Comment:
FYI: New `slurp_id_map` which preserves map of native IDs to mondo_ids used in previous slurp runs, so that multiple mondo_id's don't accidentally get assigned and used for the same terms.